### DISCUSSION TITLE
Alguien más fue retirado sin razón aparente?

### DISCUSSION THREAD

**Discussion by OATS74** [Launch-X-Latam/MisionFrontEnd#92] [General]
Buenas noches considero que he sido retirado del team sin una razon aparente.  Será momentáneo o permanente . ?

**Comment by egoz-98** [comment]
Yo sólo sé que están baneando por hacer reuniones, pero si no fue por eso, quien sabe

**Comment by Rarriaza-sv** [comment]
donde encuentro los enlaces para las clases?

**Reply by JosafatJimenezB** [reply]
Los enlaces los ponen los días que van a haber sesión, los colocan 15 minutos antes de la sesión

**Reply by CarloAyala** [reply]
Te dejo un concentrado de links: #35 
Ahi puedes ver un botón morado donde están todas las sesiones grabadas, solo la ultima no esta disponible, esa la puedes ver aquí: [Ultima Sesión](https://web.microsoftstream.com/video/41b9721d-cde1-4ffd-bdf0-7a877d57e5e1)

**Comment by LaMazorca** [comment]
Igual están babeando aquellos que comentan en las publicaciones que claramente dijeron que no deben comentar, hasta dos avisos dieron

**Reply by EduGarcia502** [reply]
aparentemente a de ser por lo mismo, siendo sinceros, <h1>hay muchos que no razonan,</h1> les vale no hacen cazo, los admin están repiten  y repiten lo mismo y siempre lo mismo, asta el momento de auto control interrumpen, no sigues las reglas, creo que muchos se gana su bam, pero bueno cada quien, el que quiera aprender le va aponer  atención  y el que no ara relajo siempre, espero no me saque por mi opinión, y disculpen sin ofender a nadie, saludos

**Comment by OATS74** [comment]
solucionado ya me ingresaron  . pero sude frio. no lo duden

**Comment by OATS74** [comment]
Hola , por donde ponen los enlaces para los live . Hace unos días entre a los live por qué estaba dentro del  equipo en ms team . Pero ahora baneado veré donde consigo el link del live. :(

Obtener Outlook para Android<https://aka.ms/ghei36>
________________________________
